storage/suffix_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"bytes"
+	"encoding/base64"
+	"math/big"
+	"testing"
+)
+
+const (
+	logStrataDepth = 8
```

Comment:
I know about the constraints, I wrote a lot of that code. If we've got constants available that can't be changed without breakage then it might be better to use them. I'll leave that decision to you.